### DISCUSSION TITLE
AuthenticationError

### DISCUSSION THREAD

**Discussion by kitsamho** [iryna-kondr/scikit-llm#23] [Q&A] [answered]
I have a working API key and Org and the library was working fine this morning but now I am getting errors when trying to use the all functionality. 

I have tried the API key directly with OpenAI and all is fine so it's not the case of a dead API key.

`Could not obtain the completion after 3 retries: `AuthenticationError :: <empty message>`
Skipping a sample due to the following error: 'NoneType' object has no attribute 'choices'`

**Comment by iryna-kondr** [comment]
Hi @kitsamho, 

Could you provide a reproducible code snippet? Also, did you use the same version of the library in the morning and right now ?

**Comment by kitsamho** [comment]
Sure - sorry

Version: 0.1.0 - and yes was the same version I am pretty sure. 

```
from skllm.config import SKLLMConfig

SKLLMConfig.set_openai_key("<working key>")
SKLLMConfig.set_openai_org("<working org>")

from skllm.preprocessing import GPTSummarizer
from skllm.datasets import get_summarization_dataset

X = get_summarization_dataset()

s = GPTSummarizer(openai_model="gpt-3.5-turbo", max_words=15)
summaries = s.fit_transform(X)
```


Error snapshot
<img width="1023" alt="Screenshot 2023-05-31 at 19 30 48" src="https://github.com/iryna-kondr/scikit-llm/assets/32895981/7d7920cc-8b2b-4664-bf19-fb0b73f5275e">

**Comment by kitsamho** [comment, marked as answer]
Ah - figured it out, for some reason I had changed the org id number back to the org name (I remembered in the readme it is ID not name! 🤦

**Comment by OKUA1** [comment]
@kitsamho, great that you solved this! It seems like using the organisation name instead of the key is a common point of confusion, so maybe we will have to rename the method itself to something like `set_openai_org_id` to be more expressive.

**Reply by kitsamho** [reply]
@OKUA1 yeah that would be helpful. Also maybe a note in the method's docstring
